lib/browser/api/screen.ts

```diff
@@ -1,38 +1,41 @@
+import { EventEmitter } from 'events';
+
 const { createScreen } = process._linkedBinding('electron_common_screen');
 
 let _screen: Electron.Screen;
 
+const ensureScreen = () => {
+  _screen = createScreen();
+  EventEmitter.call(_screen);
+};
+
 // We can't call createScreen until after app.on('ready'), but this module
 // exposes an instance created by createScreen. In order to avoid
 // side-effecting and calling createScreen upon import of this module, instead
 // we export a proxy which lazily calls createScreen on first access.
 export default new Proxy({}, {
   get: (target, prop: keyof Electron.Screen) => {
-    if (_screen === undefined) {
-      _screen = createScreen();
-    }
+    if (_screen === undefined) ensureScreen();
     const v = _screen[prop];
     if (typeof v === 'function') {
       return v.bind(_screen);
     }
     return v;
   },
   ownKeys: () => {
-    if (_screen === undefined) {
-      _screen = createScreen();
-    }
+    if (_screen === undefined) ensureScreen();
     return Reflect.ownKeys(_screen);
   },
+  getPrototypeOf: (target) => {
+    const targetPrototype = Reflect.getPrototypeOf(target);
+    return Object.assign(EventEmitter.prototype, targetPrototype);
```

Comment:
yikes, you should not be overriding things in `EventEmitter.prototype` here!!

Reply:
oops, typo. Should be swapped.

Reply:
I don't think this should edit the target's prototype, either

Reply:
@nornagon this is also what was done previously - https://github.com/electron/electron/commit/e6cf5906f60f060800d8368f80abf7019d2d4a5a#diff-3cab00a5b0a35804ff97b9c92fb7e719645bccb38c4a217957e5e58d153d5ceaL8

Reply:
@codebytere the prototype assignment is now done by C++, which is why I removed it.